jekyll/_cci2/credits.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: classic-docs
+title: Using Credits
+categories: [how-to]
+description: How to leverage CircleCI credits
+---
+
+This document describes the basics of using credits with CircleCI. If you are on the legacy CircleCI container-based plan you may want to consider consulting the document on [using containers]({{site.baseurl}}/2.0/containers). If you want to switch from using containers to using credits, please [open a support ticket](https://support.circleci.com/hc/en-us/requests/new) requesting so.
+
+## Overview
+
+The CircleCI credit-based usage plans enable you to only pay for what you use, while also providing flexibility in customizing and scaling your team's CI solution. Credits are used by the minute at varying rates according to what build configuration you use.
+
+The right plan for you and your team will depend on several factors:
+
+- How many users are on your team
+- How much support you need (community support, standard support, or premium support)
+- If you want access to different machine-types
+- If you need features such as Docker Layer Caching, concurrent builds, or build history
+
+Let's look at how a Performance plan might use credits. With the Performance plan you have:
+
+- At least 25,000 credits
+- An unlimited number of user seats (at $15 a user)
```

Comment:
This is changing in the near future when we go to market with usage plans. The upcoming logic dictates that the first three users will be provided at a lump-sum of $15, then the remainder of user seats will be provided at $15 a seat.